.circleci/config.yml

```diff
@@ -238,23 +260,23 @@ jobs:
           command: |
             bundle exec rails check_swaggerization
 
-  build_and_push:
+  build_and_push_uat:
     executor: aws-ecr/default
     steps:
-      - checkout
-      # Authenticate to AWS using OIDC v2 with the AWS CLI
-      - aws-cli/setup:
-          role_arn: $ECR_ROLE_TO_ASSUME
-          region: $ECR_REGION
-      # Authenticate to the ECR repository using the standard command
-      - run: |
-          aws ecr get-login-password --region $ECR_REGION | docker login --username AWS --password-stdin ${AWS_ECR_REGISTRY_ID}.dkr.ecr.${ECR_REGION}.amazonaws.com
-      # Build and push your Docker image
-      - aws-ecr/build-image:
-          push-image: true
-          tag: ${CIRCLE_SHA1}
-          region: $ECR_REGION # this will use the env var
-          repo: $ECR_REPOSITORY # this will use the env var
+      - build_and_push:
+          docker_tag: dev
```

Comment:
@starswan Shouldn't it be uat or production?

Reply:
no - the tags in the ECR definition are dev and staging, not UAT / staging